pkg/webhook/policy.go

```diff
@@ -92,6 +93,7 @@ func AddPolicyWebhook(mgr manager.Manager, deps Dependencies) error {
 	if err != nil {
 		return err
 	}
+	log := log.WithValues("hookType", "validation")
```

Comment:
nit: do we want to add `log.WithName("validation").WithValues...` to group the same type of logs together?